_lab/lab02.md

```diff
@@ -0,0 +1,469 @@
+---
+layout: lab
+num: lab02
+ready: true
+desc: "Tea Shop"
+assigned: 2024-01-19 19:00:00.00-7:00
```

Comment:
@krushnashah thanks! Can you please update the dates or allow maintainers to make changes?